style.css

```diff
@@ -55,23 +55,54 @@ header {
   color: inherit;
   font-weight: bold;
   left: 76px;
-  animation: color-change 1s ease-in-out;
 }
 
-@keyframes color-change {
-  0% {
-    color: rgb(247, 143, 8);
-  }
+/* start mobile-menu */
+.mobile-menu-close-icon {
+  color: #fff;
+  align-self: flex-end;
+  padding: 0;
+}
 
-  50% {
-    color: #0a5eee;
-  }
+.mobile-menu--hide {
+  display: none;
+  width: 0;
+  height: 0;
+}
 
-  100% {
-    color: rgb(83, 37, 37);
-  }
+.mobile-menu--show {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 1em 0.75em;
+  width: 100vw;
+  height: 100vh;
+  color: #fff;
+  background: rgba(96, 112, 255, 0.9);
+  font-family: "Poppins", sans-serif;
+  font-weight: 600;
+  font-size: 2rem;
+  line-height: 44px;
+  backdrop-filter: blur(4px);
+}
+
+.mobile-menu--show > ul > li {
+  padding: 0.5em 0;
 }
```

Comment:
As shown below in the figma design, the color for the menu items should be `#FFFFFF` and not `#551A4B`. They seem to inherit the color from the a tag. Kindly update it to match the actual design :+1: 

### Your design 

![Screenshot from 2023-06-19 15-04-36](https://github.com/fatima-najafi/Portfolio/assets/65233860/e51945e2-1a3e-48c6-a1eb-8666645aa0b0)

### Actual design 

![Screenshot from 2023-06-19 15-04-48](https://github.com/fatima-najafi/Portfolio/assets/65233860/9ff11173-cb3e-471e-8700-bfafc0a0ec62)